### PR TITLE
updated cats, algebra and alleycats (not released yet)

### PR DESCRIPTION
**Pull request by kailuowang**
please note that alleycats 0.1.7 is not released yet. I need to release this sbt-catalysts first to be able to release that, but if I leave the alleycats version as 0.1.6 here it will be incompatible with other dependencies. I'd rather release a temporarily not working one than something that introduces binary incomp.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ For a new project, consider using [the activator template](https://github.com/ty
 
 The current list of dependencies and versions is:
 
-+ [algebra-0.4.2](https://github.com/non/algebra)
++ [algebra-0.5.0](https://github.com/non/algebra)
 + [alleycats-0.1.3](https://github.com/non/alleycats)
```
Comment (BennyHill) [inline README.md]: Also 0.1.7 here, too

**Comment by BennyHill** [comment]
I've got a feeling that there will be a flurry of releases coming shortly when scalacheck is released, but for now :+1:

**Comment by BennyHill** [comment]
:+1:

**Comment by non** [comment]
👍

**Comment by milessabin** [comment]
:+1:

**Pull request merged**